docs/nx-cloud/enterprise/dte/azure-dte.md

```diff
@@ -32,6 +32,10 @@ jobs:
     pool:
       vmImage: 'ubuntu-latest'
     steps:
+      - checkout: self
+        fetchDepth: 1
```

Comment:
We want to have the full depth of the graph, so it would be correct to omit the `fetchDepth.`

If you set `fetchDepth: 1` then you won't be able to use `nx affected` because the base commit will be missing from your git history.

Reply:
reverted, can you check again